README.md

```diff
@@ -2,7 +2,17 @@
 
 [![Build Status](https://jenkins-etcd-public.prod.coreos.systems/buildStatus/icon?job=etcd-operator-master)](https://jenkins-etcd-public.prod.coreos.systems/job/etcd-operator-master/)
 
-**Project status: *alpha*** Not all planned features are completed. The API, spec, status and other user facing objects are subject to change. We do not support backward-compatibility for the alpha releases.
+## Project status
+
+etcd operator is a **beta** software.
+
+Major planned features are completed. The etcd cluster spec, status are fairly stable. Any breaking changes of user facing objects or API will be pointed out in release notes. 
```

Comment:
After a second thought, I think we could better phrase that:

Major planned features are completed. Our API is beta. We don't intend to break any API unless we found out any mistake before stable. We will announce the changes to API in release notes.